packages/client/foreman_ygg_migration/foreman_ygg_migration.spec

```diff
@@ -0,0 +1,34 @@
+Name: foreman_ygg_migration
+Version: 0.0.1
+Summary: A helper package to ease transition from yggdrasil 0.2.z to 0.4.z
+Release: 1%{?dist}
+License: MIT
+
+%if 0%{?rhel} >= 8
+Supplements: yggdrasil >= 0.4.0
```

Comment:
People can (and do :cry:) disable weak dependencies.
For such setups it will mean that this package won't be automatically installed once they upgrade to 0.4.

This is not a blocker (and there is no better package relationship to fix this), but something we should keep in mind and document somewhere.

Reply:
What exactly should we document and where?

Reply:
We should document that if people have weak dependencies disabled, they should install this package explicitly. As to where, I'm not sure if we already have a procedure or something where this would fit well or we'll need to write a new one

Reply:
Yeah, I am not aware of a good place for this (maybe at least release notes?)